src/org/opendatakit/briefcase/util/FileSystemUtils.java

```diff
@@ -74,11 +78,46 @@
   static final String ENCRYPTED_FILE_EXTENSION = ".enc";
   static final String MISSING_FILE_EXTENSION = ".missing";
 
+  static final File cacheFile = new File(getBriefcaseFolder(), "cache.ser");
+
+  static Map pathToHashMap;
+  static Map pathToDefinitionMap;
+
   public static final String getMountPoint() {
     return System.getProperty("os.name").startsWith("Win") ? File.separator + ".." : (System
         .getProperty("os.name").startsWith("Mac") ? "/Volumes/" : "/mnt/");
   }
 
+  public static void saveFormDefinitionCache() {
+    if (!cacheFile.exists() || cacheFile.canWrite()) {
+      try (ObjectOutputStream objectOutputStream = new ObjectOutputStream(new FileOutputStream(cacheFile))) {
+        objectOutputStream.writeObject(pathToHashMap);
+        objectOutputStream.writeObject(pathToDefinitionMap);
+      } catch (IOException e) {
+        e.printStackTrace();
+      }
+    }
+  }
+
+  public static void loadFormDefinitionCache() {
```

Comment:
Need to add warning suppression to this method (whether Maps are parameterized or not).